presto-hive/src/main/java/io/prestosql/plugin/hive/orc/OrcFileWriter.java

```diff
@@ -143,6 +170,11 @@ public void appendRows(Page dataPage)
     public void commit()
     {
         try {
+            if (transaction.isAcidTransactionRunning() && useAcidSchema) {
+                int bucketValue = computeBucketValue(bucketNumber.isEmpty() ? 0 : bucketNumber.getAsInt(), 0);
+                long writeId = maxWriteId.isPresent() ? maxWriteId.getAsLong() : transaction.getWriteId();
```

Comment:
why not always take writeId from transaction?

Reply:
I wanted to leave open the possibility that the OrcFileWriter wrote on behalf of a range of writeids.  This will be needed when we support multi-operation transactions.